src/components/App/App.js

```diff
@@ -13,25 +13,21 @@ import './App.scss';
 class App extends Component {
 	constructor() {
 		super();
-
 		this.state = {
-			currentUser: '',
-			id: '',
+			currentUser: {},
```

Comment:
Great that this is an object now with all of the user's data. Makes it easier to pass around the data as we need to without having to expand extra room in state for it.